pkg/catalog/generator.go

```diff
@@ -77,24 +86,25 @@ func (p *Generator) SourcePath(a SourceCoder) string {
 
 // NewProject generates a Generator object, fs and outputDir are optional if being used for a web server.
 func NewProject(
-	titleAndFileName, plantumlService, outputType string,
-	log *logrus.Logger,
+	titleAndFileName, plantumlService, outputType, feedbackLink, chatLink string,
```

Comment:
This could be a good opportunity to refactor NewProject to take in a ProjectConfig struct, with each of these as fields. No worries if you don't want to make it part of this PR

Reply:
Open an issue for this: https://github.com/anz-bank/sysl-catalog/issues/75.